### ISSUE TITLE
RNG behavior in parallel environment

### ISSUE THREAD

**Issue by rmandelb** [GalSim-developers/GalSim#67]
As Mike pointed out in issue #32, we should figure out how the RNG will behave in a parallel environment, since it is very likely we'll want to be producing galaxies in parallel at some point.  I was not sure how to carry out the test he described, but wanted to have simple RNG tests in place, so I am moving this more complex test to its own issue.

Mike, would you be willing to take this on after the first milestone is done?  (Or, provide guidance to Barney or me so we can carry it out?)
